roles/openshift_cloud_provider/tasks/vsphere.yml

```diff
@@ -3,4 +3,26 @@
   template:
     dest: "{{ openshift.common.config_base }}/cloudprovider/vsphere.conf"
     src: vsphere.conf.j2
-  when: openshift_cloudprovider_vsphere_username is defined and openshift_cloudprovider_vsphere_password is defined and openshift_cloudprovider_vsphere_host is defined and openshift_cloudprovider_vsphere_datacenter is defined and openshift_cloudprovider_vsphere_datastore is defined
+  when: 
+  - openshift_cloudprovider_vsphere_username is defined 
+  - openshift_cloudprovider_vsphere_password is defined 
+  - openshift_cloudprovider_vsphere_host is defined 
+  - openshift_cloudprovider_vsphere_datacenter is defined 
+  - openshift_cloudprovider_vsphere_datastore is defined
+
+- name: configure vsphere svc account
+  include_role:
```

Comment:
Don't need include_role, you're in the same role.

Use "import_tasks"

Reply:
Done